### DISCUSSION TITLE
run fatal

### DISCUSSION THREAD

**Discussion by akin520** [FerretDB/FerretDB#129] [General]
![image](https://user-images.githubusercontent.com/1317859/145182286-0e219ae9-9faa-402d-b648-3849c361e315.png)
![image](https://user-images.githubusercontent.com/1317859/145182367-901f49a4-387d-4f8e-804e-f1344f5b6bd6.png)

**Comment by AlekSi** [comment]
The current workaround is to switch both collate and ctype to `en_US.utf8` (without a dash).
It will be fixed in the next version; see https://github.com/FerretDB/FerretDB/issues/130.

**Reply by akin520** [reply]
![image](https://user-images.githubusercontent.com/1317859/145186711-9c6a1101-2049-4b04-b82a-7d9aa2a2bf76.png)
RUN OK
